src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -148,4 +158,38 @@ export class AppConfig extends AbstractAppConfig {
   public getWorkAllocationApiUrl(): string {
     return this.workallocationUrl;
   }
+
+  public getHrsUrl(): string {
+    return '';
+  }
+
+  public getRemoteHrsUrl(): string {
+    return '';
+  }
+
+  public prependedCaseViewTabs(): Observable<CaseTab[]> {
+    return combineLatest([
+      this.featureToggleService.getValue(AppConstants.FEATURE_NAMES.currentWAFeature, 'WorkAllocationRelease1'),
+      this.store.pipe(select(fromRoot.getUserDetails))
+    ]).pipe(
+      map(([feature, userDetails]: [string, UserDetails]) => this.enablePrependedTabs(feature, userDetails) ? this.tabs : [])
+    );
+  }
+
+  private featureToggleWorkAllocation(): void {
+    this.featureToggleService
+      .isEnabled(AppConstants.FEATURE_NAMES.workAllocation)
+      .subscribe(
+        (isFeatureEnabled) =>
+          this.workallocationUrl = AppUtils.getFeatureToggledUrl(
+            isFeatureEnabled,
+            WorkAllocationTaskService.WorkAllocationUrl
+          )
+      );
+  }
```

Comment:
Could these move to the container(exui container) level? 
The problem to put here is ccd-case.config.ts is intantiated/implemented when the APP is started, we shouldn't put the container's dynamic logic here.
The ccd-case.config.ts is used for put all those configuration(constant immutable value) settings.

Reply:
@rayliangatos I cannot move this to container level. As this config needs to be passed to the `toolkit`. As well as its according to the design. 
https://tools.hmcts.net/confluence/display/EUI/CCD+Toolkit+-+Custom+added+tabs